proto/penumbra/penumbra/core/component/dex/v1alpha1/dex.proto

```diff
@@ -561,3 +561,55 @@ message SimulateTradeRequest {
 message SimulateTradeResponse {
   core.component.dex.v1alpha1.SwapExecution output = 1;
 }
+
+message EventSwap {
+  // The trading pair to swap.
+  TradingPair trading_pair = 1;
+  // The amount for asset 1.
+  num.v1alpha1.Amount delta_1_i = 2;
+  // The amount for asset 2.
+  num.v1alpha1.Amount delta_2_i = 3;
+  // A commitment to a prepaid fee for the future SwapClaim.
```

Comment:
Why do we care about this in the event? Indexers probably don't care about it at all right, there's no reason to have this information...

It does seem useful to have the _swap commitment_, so that it's possible to query the event DB by swap commitment?